tutor

```diff
@@ -0,0 +1,2 @@
+Dodaje jakis tekst.'A' dodaje znak na koncu linii.
```

Comment:
Hmm... what about the vimtutor exercise? I want to grant you some points for it, but I need evidence ;)

Reply:
Hi Łukasz, I decided to stay on the ':wq' level of VIM knowledge :D 
I did the tutor 10, 5 years ago - no benefit so far. 
I had only one use case in my life - fix some settings on remote raspberry pi 0 prototype of something. I accept reduced points. 
Course is great so far! Regards!

Reply:
Thanks! So no points for vim ;)